.github/CODEOWNERS

```diff
@@ -116,6 +116,8 @@
 
 /src/account/ @zikalino
 
+/src/azext_quantum/ @anpaz-msft
```

Comment:
The naming style is different from others.

Reply:
that was auto-generated. fixed.